test/js/upload-wizard/upload-new-study.test.js

```diff
@@ -1,60 +1,48 @@
-import React from 'react'
-import { render, screen, fireEvent, waitForElementToBeRemoved } from '@testing-library/react'
+import { screen, fireEvent, waitForElementToBeRemoved } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
 import _cloneDeep from 'lodash/cloneDeep'
 import selectEvent from 'react-select-event'
-import ReactNotification from 'react-notifications-component'
 
-import { RawUploadWizard } from 'components/upload/UploadWizard'
-import MockRouter from '../lib/MockRouter'
 import { fireFileSelectionEvent } from '../lib/file-mock-utils'
 import * as ScpApi from 'lib/scp-api'
 import {
-  EMPTY_STUDY, RAW_COUNTS_FILE, PROCESSED_MATRIX_FILE, METADATA_FILE,
-  CLUSTER_FILE
+  RAW_COUNTS_FILE, PROCESSED_MATRIX_FILE, METADATA_FILE,
+  CLUSTER_FILE, COORDINATE_LABEL_FILE, FASTQ_FILE
 } from './file-info-responses'
-import { UserContext } from 'providers/UserProvider'
-
-/** gets a pointer to the react-select node based on label text
- * This is non-trivial since our labels contain the select,
- * and so a naive getByLabelText will not work.
- * Instead we get the label using getByText and assume
- * the first div inside is the react-select element */
-function getSelectByLabelText(screen, text) {
-  return screen.getByText(text).querySelector('div')
-}
+import { renderWizardWithStudy, getSelectByLabelText, saveButton } from './upload-wizard-test-utils'
+
 
 describe('creation of study files', () => {
-  it('renders the raw counts form and allows updates and save', async () => {
-    const studyInfoSpy = jest.spyOn(ScpApi, 'fetchStudyFileInfo')
-    // pass in a clone of the response since it may get modified by the cache operations
-    studyInfoSpy.mockImplementation(params => {
-      const response = _cloneDeep(EMPTY_STUDY)
-      return Promise.resolve(response)
-    })
+  beforeAll(() => {
+    jest.restoreAllMocks()
+    // This test is long, so allow extra time
```

Comment:
```suggestion
    // This test is long, as it uploads in sequence like users would.  So allow extra time.  
```

Thanks for clarifying the intent, and thus why the tests run long.  Briefly documenting that here will (one hopes) avoid setting off my alarm bells next time I see this.